### PR TITLE
Handle basics lasers and reflections

### PR DESCRIPTION
**Pull request by Bruhtek**
It should work properly now.
Still doesn't handle refractions and losses of intensity - that will be done later.

Ref: #10

### CODE REVIEW

**Comment by Bruhtek** [comment]
Still doesnt work properly yet actually

File: src/utils/geometry (conflicted copy 2024-02-07 091846).ts
Comment (MrJujek) [inline src/utils/geometry (conflicted copy 2024-02-07 091846).ts:1]: delete this

**Review by MrJujek — CHANGES_REQUESTED** [review]
delete geometry (something copy numbers).ts

**Comment by Bruhtek** [comment]
Should hopefully cover all cases now.

**Review by MrJujek — APPROVED** [review]
gj

**Pull request merged**